packages/block-editor/src/hooks/use-bindings-attributes.js

```diff
@@ -296,9 +296,18 @@ export const withBlockBindingSupport = createHigherOrderComponent(
 			</>
 		);
 	},
-	'withBlockBindingSupport'
+	'editWithBlockBindings'
 );
 
+function labelWithBlockBindings( settings ) {
+	return ( attributes, { context } ) => {
+		if ( attributes?.metadata?.bindings && context === 'accessibility' ) {
+			return 'Overriding label';
```

Comment:
> For now, I just check whether the paragraph is bound and, if so, return the label `Paragraph with dynamic content`.

Is `Overriding label` a leftover added for testing? In any case, it should be translatable.

Reply:
Thanks for the feedback! Rather than hardcoding this, I've now modified the PR so it reads the actual bindings values. I've updated the PR description accordingly.